### PR TITLE
add bpm related files back

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/learn/adding-a-workflow-engine.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Adding a Workflow Engine
+
+This section demonstrates how to configure the WSO2 Identity Server (IS)
+to connect to the [Business Process Management (BPM) component of WSO2
+Enterprise
+Integrator](https://docs.wso2.com/display/EI650/Working+with+Business+Processes)
```
Comment (ayshsandu) [inline en/docs/learn/adding-a-workflow-engine.md]: Please mention the compatible EI version visibly in the document

File: en/docs/learn/adding-a-workflow-engine.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Adding a Workflow Engine
+
+This section demonstrates how to configure the WSO2 Identity Server (IS)
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: ... how go configure WSO2 Identity Server ...

File: en/docs/learn/adding-a-workflow-engine.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+and use one of them when creating a workflow.
+
+!!! info 
+    An embedded Business Process Management engine is included in WSO2
+    Identity Server. However, if you need more flexibility and more
+    extensibility with this feature than what is currently provided, it is
+    advisable to [integrate the Business Process Management (BPM) component
+    of WSO2 Enterprise
+    Integrator](../../learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine)
+    with the Identity Server.
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: WSO2 Identity Server

File: en/docs/learn/adding-a-workflow-engine.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+1.  To create a BPM profile click on **Configure \> Workflow Engine
+    Profiles \> Add**. The window below will be displayed.
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: To create a BPM profile, .....

Please follow the guidelines in https://docs.google.com/document/d/14dZtE-f0BF46t5woX9k5aOawN1aIJ4WKaIFRqe7VyP8/edit#heading=h.i7s3q1sgw4a

File: en/docs/learn/adding-a-workflow-engine.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+2.  Fill in the details using the descriptions below to guide you and
+    click **Add**.  
+    ![creating-a-bpm-profile](../assets/img/using-wso2-identity-server/creating-a-bpm-profile.png)
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: Please follow the guidelines in https://docs.google.com/document/d/14dZtE-f0BF46t5woX9k5aOawN1aIJ4WKaIFRqe7VyP8/edit#heading=h.i7s3q1sgw4a

File: en/docs/learn/adding-a-workflow-engine.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    | BPS Worker Host Services URL  | The host URL of the BPS worker node.                                                                                                                                                                                                              | `                               https://localhost:9444/services                             ` (This points to the worker node of WSO2 BPS that is already running.)  |
+    | Username                      | The username that is used to authenticate the user in WSO2 BPS. The requests are sent to BPS as this user. Therefore, the user needs to have permission to deploy and invoke Business Process Execution Languages (BPEL) and perform human tasks. | `               user1              `                                                                                                                                 |
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: user name

File: en/docs/learn/adding-a-workflow-engine.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    | Password                      | The password of the user mentioned above.                                                                                                                                                                                                         | `               passwd              `                                                                                                                                |
+
+### What's next?
+
+-   Need more flexibility and more extensibility with this feature?
+    [Integrate the Business Process Management (BPM) component of WSO2
+    Enterprise Integrator with the Identity
+    Server](../../learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine).
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: To have more flexibility and extensibility in this feature, see []().

File: en/docs/learn/adding-a-workflow-engine.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+-   Now you need to create a new workflow definition. For more
+    information, see [Adding a New Workflow
+    Definition](../../learn/adding-a-new-workflow-definition).
```
Comment (YvonneW) [inline en/docs/learn/adding-a-workflow-engine.md]: To create a new workflow definition, see []().

File: en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+!!! info 
+    The `<EI_HOME>/wso2/business-process` is referred to as `<BPM_HOME>` throughout this document.
+
+1.  Create a separate database to store the user information using the relevant DB schema. For example, you can execute the following DB scripts to create a MYSQL database.  
+
+    ```sql
+    <IS_HOME>/dbscripts/mysql.sql
+    <IS_HOME>/dbscripts/identity/mysql.sql
+    ```
+2.  Configure the user management database in both the IS and the BPM server. Add the following configurations with appropriate values in both the servers. Make sure that the URLs mention the name of the database you created in the previous step and the username and password are corresponding to your MySQL instance.
```
Comment (YvonneW) [inline en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md]: IS -> WSO2 Identity Server

Comment (YvonneW) [reply en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md]: user name

File: en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+    ```curl
+    cd <IS_HOME>/bin 
+    ./wso2server.sh
+    ```
+    
+    ```curl 
+    cd <EI_HOME>/bin
+    ./business-process.sh
+    ```
```
Comment (YvonneW) [inline en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md]: Provide commands for Windows users as well OR point to a place where those commands are presented.

File: en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+7.  To test if the BPM profile has been configured properly to the Identity Server, create a user in the WSO2 Identity Server Management Console. Now when you navigate to the list of users in Business Process Server Management Console, the user created previously should be present.
```
Comment (YvonneW) [inline en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md]: To test if the BPM profile is properly configured in WSO2 Identity Server,

Comment (YvonneW) [reply en/docs/learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md]: Better to break this to multiple sub-points. 

Guidance: https://docs.google.com/document/d/14dZtE-f0BF46t5woX9k5aOawN1aIJ4WKaIFRqe7VyP8/edit#heading=h.i7s3q1sgw4a

File: en/docs/setup/installing-as-a-windows-service.md [hunk 12]
```diff
@@ -22,7 +22,7 @@ Follow the instructions in the sections below to set it up.
 -   Download and install a service wrapper library for running
     your WSO2 product as a Windows service. WSO2 recommends _Yet Another
```
Comment (YvonneW) [inline en/docs/setup/installing-as-a-windows-service.md]: WSO2 Identity Server

File: en/mkdocs.yml [hunk 13]
```diff
@@ -287,6 +287,9 @@ nav:
           - 'Setting Up ReCaptcha': learn/setting-up-recaptcha.md
         - 'Workflows': 
           - 'Workflows': learn/workflow-management.md
+          - 'Adding a workflow Engine':
+            - 'Adding a workflow Engine': learn/adding-a-workflow-engine.md
+            - 'Configuring the BPM Profile as a Workflow Engine': learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md
```
Comment (YvonneW) [inline en/mkdocs.yml]: 'Configuring BPM as a Workflow Engine' ?

Comment (gomathyk) [reply en/mkdocs.yml]: BPM has been deprecated now. So I feel that emphasizing that it is only a profile helps.

**Pull request merged**